src/js/components/Notification/__tests__/Notification-test.js

```diff
@@ -263,4 +264,31 @@ describe('Notification', () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  test('should render custom icon', () => {
+    const { asFragment } = render(
+      <Grommet>
+        <TestNotification icon={<Home />} />
+      </Grommet>,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  test('should render the default icon if no icon is passed', () => {
+    const theme = {
+      notification: {
+        unknown: {
+          icon: <Home />,
+          color: 'blue',
+        },
+      },
+    };
+    const Test = () => {
+      <Grommet theme={theme}>
+        <Notification data-testid="test" title="Test title" message="message" />
+      </Grommet>;
+    };
+    const { asFragment } = render(<Test />);
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
It looks like the snapshot isn't showing the notification as expected for this test. I think if we change this to the following the test will work

```suggestion
    const theme = {
      notification: {
        unknown: {
          icon: Home,
          color: 'blue',
        },
      },
    };
    const { asFragment } = render 
      <Grommet theme={theme}>
        <Notification title="Test title" message="message" />
      </Grommet>;
    };

    expect(asFragment()).toMatchSnapshot();
```

Reply:
Thanks @jcfilben , made the changes and tested on local. New Snapshot is being created. :)